packages/smooth_app/lib/cards/product_cards/smooth_product_card_found.dart

```diff
@@ -38,31 +40,19 @@ class SmoothProductCardFound extends StatelessWidget {
       return _getOldStyle(context);
     }
 
-    final ProductPreferences productPreferences =
-        context.watch<ProductPreferences>();
     final Size screenSize = MediaQuery.of(context).size;
     final ThemeData themeData = Theme.of(context);
 
-    final List<String> attributeIds =
-        productPreferences.getOrderedImportantAttributeIds();
     final List<Widget> scores = <Widget>[];
     final double iconSize = IconWidgetSizer.getIconSizeFromContext(context);
     final List<Attribute> attributes =
-        AttributeListExpandable.getPopulatedAttributes(product, attributeIds);
+        AttributeListExpandable.getPopulatedAttributes(
+            product, <String>['nutriscore', 'ecoscore']);
```

Comment:
This is the same as summary_card.dart L:19. Maybe this constant can be shared in `ui_helpers.dart`